.github/workflows/coverage-orchestrator.yml

```
@@ -0,0 +1,26 @@
+
+jobs:
+  run-cmake:
+    uses: ./.github/workflows/cmake.yml
+    secrets: inherit
+
+  run-pigz:
+    uses: ./.github/workflows/pigz.yml
+    secrets: inherit
+
+  # This job only starts if ALL reusable workflows above succeed
+  final-upload:
```

actionlint (1.7.11)

[error] job "final-upload" needs job "run-lint" which does not exist in this workflow

(job-needs)